web_scraping/BaseScraper.py

```diff
@@ -141,6 +141,7 @@ def insert_rows(self, df):
         row_insertions = []
         for i in list(df.itertuples(index=False)):
             row_insertions.append((
+                  int(i.review_id.strip("rw")),
                   i.movie_id,
                   i.date,
                   int(i.rating),
```

Comment:
casting rating to an int -- how will that work with letterboxd ratings which are floats .5-5.0?

Reply:
@ekoly

Reply:
@moviedatascience good catch. Fixed, and added divide by 2